src/devtools/devtools.tsx

```diff
@@ -494,7 +510,7 @@ export const ReactQueryDevtoolsPanel = React.forwardRef<
             maxHeight: '100%',
             overflow: 'auto',
             borderRight: `1px solid ${theme.grayAlt}`,
-            display: 'flex',
+            display: isOpen ? 'flex' : 'none',
```

Comment:
This change causes one test to fail. I'm investigating.
```bash
useInfiniteQuery › should render correct states even in case of concurrent renders with different properties
```